tools/extras/install_mkl.sh

```diff
@@ -254,6 +254,16 @@ ConfigLdSo() {
     ldconfig )
 }
 
+(
+  set +u
+
+  echo "export LD_PRELOAD:=/opt/intel/mkl/lib/intel64/libmkl_def.so:\
```

Comment:
I'll merge this, but I'd like someone to remind me why we are using LD_PRELOAD and not LD_LIBRARY_PATH.

Reply:
Not sure. LD_LIBRARY_PATH just doesn't work...

Reply:
so you did something like "export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/intel/mkl/lib/intel64/" ?

Reply:
Yes, it doesn't work.

Reply:
wait, let me try again

Reply:
yep. double checked, doesn't work.

Reply:
OK.  MKL is weird.  Document the +u, why it's needed and what it does, and I'll merge.

Reply:
oh, you removed it..